### PR TITLE
Documentation Improvement: Checking versions of prerequsites

### PR DESCRIPTION
**Pull request by utsab**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX
#### Description

This PR improves documentation. For people who already have pre-existing versions of node and mongo, provide instructions for making sure they have the right versions. This can help prevent contributors from experiencing incompatibility errors.

### CODE REVIEW

**Review by systimotic — COMMENTED** [review]
👍 I like this change. A lot of people seem to miss this and still try to run FCC with Node v4. Maybe this will help. It seems like you indented the commands a bit. Everywhere else in this document they're right up against the left edge. I think it would be a good idea to remove the indentation here. Maybe the sentence inside the brackets should be expanded to a full sentence below the code block, saying that if they're lower than the required versions, they should update.

**Review by systimotic — APPROVED** [review]
LGTM! 🎉

**Comment by utsab** [comment]
@systimotic Thanks for the feedback.  We factored it in and added a 2nd commit to this PR.  Question: I know the best practice is to have only a single commit per PR.   Should we squash the commits somehow?  This is our first PR, so not sure how to do this.

**Comment by systimotic** [comment]
Yesss please! You can [find instructions on how to squash your commits here](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231)

**Comment by utsab** [comment]
Thanks @systimotic, I squashed the commits .

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,6 +41,16 @@ Remember to feel free to ask for help in our [Contributors](https://gitter.im/Fr
...
+If Node or MongoDB is already installed in your machine, run the following commands to validate the versions:
+
+```shell 
+node -v 
+mongo --version
+``` 
+
+If your versions are lower than the prerequisite versions, you should update. 
+
+
```
Comment (atjonathan) [inline CONTRIBUTING.md]: Remove this line

**Review by ghost — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Bouncey) [inline CONTRIBUTING.md]: I think @atjonathan meant 'Remove this extra line (53)' :+1:

**Comment by utsab** [comment]
@atjonathan, I removed the extra newline as you requested.

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]
#### LGTM 👍

Thanks for the contribution!

**Pull request merged**